src/Angor/Client/Pages/Wallet.razor

```diff
@@ -849,31 +945,44 @@ else
     {
         if (string.IsNullOrEmpty(newWalletWords))
         {
-            walletWordsCreateModal = false;
-            notificationComponent.ShowErrorMessage("New wallet words are null or empty");
+            //walletWordsCreateModal = false;
+            notificationComponent.ShowErrorMessage(
+                isNewWallet
+                ? "You need to generate your wallet words before continuing."
+                : "Please enter or paste your wallet words to recover your wallet."
+            );
             return;
         }
 
         if (string.IsNullOrEmpty(newWalletPassword))
         {
-            walletWordsCreateModal = false;
-            notificationComponent.ShowErrorMessage("New wallet password is null or empty");
+            //walletWordsCreateModal = false;
```

Comment:
ah just saw this, is it intentionally commented out?

Reply:
Yes, I don't want the modal to close when an error is displayed.